Helpers/SecurityActionHelpers.cs

```diff
@@ -0,0 +1,25 @@
+namespace Cliptok.Helpers
+{
+    public class SecurityActionHelpers
+    {
+        public static async Task<JToken> GetCurrentSecurityActions(ulong guildId)
+        {
+            using HttpRequestMessage getActionsRequest = new(HttpMethod.Get, $"https://discord.com/api/v9/guilds/{guildId}");
```

Comment:
Lets make this dynamic so when we do upgrade API version they're all done at the same time.
```suggestion
            using HttpRequestMessage getActionsRequest = new(HttpMethod.Get, $"https://discord.com/api/v{Program.discord.GatewayVersion}/guilds/{guildId}");
```